_amenities/pools/swim-lessons.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: default
+title: Swim Lessons
+date: 2019-06-10 06:01
+author: Klahanie
+comments: true
+categories: [pool]
+show-title: true
+---
+
+<div class="alert alert-warning">
+<strong>Note</strong> Klahanie Pools and swim lessons are currently closed due to the COVID-19 pandemic. 
+</div>
+
+<div class="card float-right ml-4 mb-4">
+  <div class="card-header">
+    Quick links
+  </div>
+  <ul class="list-group list-group-flush">
+    <li class="list-group-item"><a href="https://www.teamunify.com/cmkhoaa">TeamUnify</a></li>
+    <li class="list-group-item"><a href="{{site.url}}files/liability-waiver.pdf">Liability Waiver</a></li>
+    <li class="list-group-item"><a href="{{site.url}}files/cancellation-policy.pdf">Cancellation policy</a></li>
+    <li class="list-group-item"><a href="mailto:swimlessons@klahanie.com">Swim Lesson Coordinator</a></li>
+    <li class="list-group-item"><a href="tel:425.463.6388">Call the office at (425) 463-6388</a></li>
+  </ul>
+</div>
+
+### Off-season group lessons
+
+Off-Season Group Swim Lesson sessions run 1 class per week for 4 weeks. There are a variety of times available. Lessons are available for both Klahanie and non-Klahanie residents.
+
+* Swim lessons are $32 per child for Klahanie residents and $46 per child for non-Klahanie residents, per session (4 lessons total).
+  - Lessons are 30 minutes long, 1 class per week for 4 weeks.

```

Comment:
A bullet should not be used when there is only one item listed. Move this info up?